src/main/java/appeng/core/Registration.java

```diff
@@ -150,18 +143,11 @@ void preInitialize( final FMLPreInitializationEvent event )
 
 		MinecraftForge.EVENT_BUS.register( OreDictionaryHandler.INSTANCE );
 
-		final ApiDefinitions definitions = api.definitions();
+		ApiDefinitions definitions = api.definitions();
```

Comment:
Is it part of the AE style guide to make EVERYTHING final? I've seen it around the code base on and off, but I don't know why it's being done to that extent.